.github/workflows/main.yml

```diff
@@ -73,30 +82,25 @@ jobs:
         run: |
           pip install --progress-bar off torch==${{ matrix.torch-version }}
           pip install --progress-bar off tensorflow==${{ matrix.tensorflow-version }}
-      - name: Install Texar
-        run: |
-          git clone https://github.com/asyml/texar-pytorch.git
-          cd texar-pytorch
-          pip install --use-feature=in-tree-build --progress-bar off .
-          cd ..
-          # Remove them to avoid confusion.
-          rm -rf texar-pytorch
       - name: Install Forte
         run: |
           pip install --use-feature=in-tree-build --progress-bar off .[models,test,wikipedia,data_aug,nlp,ir,texar-encoder,stave,audio_ext,remote,extractor]
+      - name: Test models module
+        run: |
+          pip install --use-feature=in-tree-build --progress-bar off .[test]
+          if [ ${{ matrix.modules.module }} != "" ]
+          then
+            pip install --use-feature=in-tree-build --progress-bar off .[${{ matrix.modules.module }}]
+            pip install --progress-bar off pytest==5.1.3 coverage
+          fi
       - name: Test backbone Forte import test
         run: |
           # Try to install Forte backbone only and test basic imports.
           pip install --use-feature=in-tree-build --progress-bar off .
           # needs to remove it in after torch dependency is removed
           pip uninstall -y texar-pytorch
```

Comment:
I recalled you mentioned this can be deleted? Or do we need to delete it after removing "torch

Reply:
Since the test import only test nondependency on texar but it still has dependency on `torch`, `torchtext` etc. 
Therefore, there are two ways for testing non dependency:
1. Remove install/uninstall script and also remove test importings that have dependency on other extra packages such as torch, torchtext and transformers.
2. test non dependency for each package independently just as this script with install/uninstall script.

Reply:
what i don't understand is why `texar-pytorch` appears here, shouldn't we  test this after we install Forte only? Before installing any extra dependencies?

Reply:
The test file `texar_nondependency_test.py` is only testing `texar`'s nondependency but those classes in the test file still have dependencies on other extra required packages. 
- install forte
- install all extra required packages
- uninstall texar and test `texar`'s nondependency and install texar back

Reply:
If we only keep classes have dependencies on forte in `texar_nondependency_test.py`, then we can have pipeline 
- install forte
- test nondependency for all extra required packages
- install required packages to test packages separately (dependency matrix)
- install all extra required packages and test all files (regular testing)

Reply:
Which extra requires packages are needed? We should only test the backbone Forte here, so it shouldn't require other extra requires.

Reply:
It requires `torch`, `transformers` and `torchtext` from the previous output. I can check and remove classes actually use them.
The other way is to build up nondependency one by one as current install/uninstall pipeline and finally have a backbone test.